eng/testing/tests.mobile.targets

```
@@ -211,7 +211,7 @@
         <CultureName>$([System.IO.Directory]::GetParent('%(Identity)').Name)</CultureName>
       </WasmSatelliteAssemblies>
 
-      <WasmAssembliesToBundle Include="$(PublishDir)\*.dll"/>
+      <WasmAssemblies Include="$(PublishDir)\*.dll"/>
```

Bit of a nit because I see this convention broken far too often, but it's common to use the singular case when naming items. So this would be

```suggestion
      <WasmAssembly Include="$(PublishDir)\*.dll"/>
```